scripts/rollup/build.js

```diff
@@ -150,6 +150,12 @@ function updateBundleConfig(config, filename, format, bundleType, hasteName) {
   });
 }
 
+function setUseFiberEnvVariable(useFiber) {
+  return {
+    'process.env.ROLLUP_USE_FIBER': useFiber,
```

Comment:
Oh. Maybe I misunderstood what you were suggesting them (and I actually don't know after all). Could you give a more concrete example?